### ISSUE TITLE
PlayerGameLogs appears to return 2016-17 data regardless of what's passed in

### ISSUE THREAD

**Issue by jredwards** [swar/nba_api#99]
Doesn't appear to matter how the call is structured, whether you're filtering to a player, a team, or trying to filter to a season. The team parameter works. The player parameter works. The season parameter does not work and the API call returns data from 2016-17 always.

This is specifically the new playergamelogs (plural) endpoint that was added this past weekend.

**Comment by jredwards** [comment]
I guess I should add: as far as I can tell, this is an issue with the API endpoint itself and not this library. This may be an obvious close, but I thought it was worth pointing out that this was happening.

**Comment by swar** [comment]
What code are you trying to use?

I am able to get it to work via Browser for different seasons: https://stats.nba.com/player/2544/boxscores-advanced/?Season=2012-13&SeasonType=Regular%20Season

**Comment by jredwards** [comment]
That's boxscores advanced. I'm specifically referring to this endpoint:
https://github.com/swar/nba_api/blob/master/docs/nba_api/stats/endpoints/playergamelogs.md

Using the library you can make a call like: `playergamelogs.PlayerGameLogs(player_id_nullable=203076)`
or
`playergamelogs.PlayerGameLogs(team_id_nullable=1610612737)`
or
`playergamelogs.PlayerGameLogs(team_id_nullable=1610612737,season_nullable='2015-16')` (I'm actually not sure if that's how season is supposed to be passed. I also tried passing season IDs)

Regardless of how you make the call, you'll get 2016-17 data.

**Comment by swar** [comment]
I figured out the issue. The endpoint for that page says that "Season" should be "SeasonYear", unfortunately it only accepts "Season" as a valid parameter. nba_api is assuming it is SeasonYear because that's what the response is indicating. You can work around this issue by doing the following:
```python
raw_data = playergamelogs.PlayerGameLogs(player_id_nullable=2544, get_request=False)
raw_data.parameters['Season'] = '2012-13'
raw_data.get_request()
```

This will be marked as a bug and fixed in an upcoming release. Just need to find some free time.

**Comment by d-malko** [comment]
Fixed this issue and added PR.

**Comment by swar** [comment]
The fix needs to be implemented in the tool that analyzes endpoints instead of the file because this fix is just temporary and will be overwritten during the next version update prior to analyzing the endpoints.

**Comment by swar** [comment]
Fixed as of [v1.1.8](https://github.com/swar/nba_api/releases/tag/v1.1.8).
